apps/mobile/src/features/send/send-form/validation/btc.validation.ts

```diff
@@ -0,0 +1,37 @@
+import { t } from '@lingui/macro';
+
+import { BitcoinErrorKey, minSpendAmountInSats } from '@leather.io/bitcoin';
+import { match, satToBtc } from '@leather.io/utils';
+
+const minimumBtcSpend = satToBtc(minSpendAmountInSats);
+
```

Comment:
We need to map the error key to a lingui crowdin key. 

I wonder should we investigate if we can move this into `packages/bitcoin` / somewhere shared and then consume it from the extension to have translated errors